runtime/shared_common/CacheMap.cpp

```diff
@@ -5191,6 +5191,14 @@ SH_CacheMap::printCacheStatsTopLayerStatsHelper(J9VMThread* currentThread, UDATA
 		CACHEMAP_PRINT1(J9NLS_DO_NOT_PRINT_MESSAGE_TAG, J9NLS_SHRC_CM_PRINTSTATS_FEATURE, "default");
 	}
 
+	j9tty_printf(_portlib, "\t");
+	J9SharedDataDescriptor firstDescriptor;
+	firstDescriptor.address = NULL;
+	findSharedData(currentThread, "J9AOTHeader", sizeof("J9AOTHeader") - 1, J9SHR_DATA_TYPE_AOTHEADER, FALSE, &firstDescriptor, NULL);
+	char buff[500];
```

Comment:
I guess line 5194 to line 5198 can all be moved into printAOTHeader().

Reply:
I tried to do `findSharedData()` in `printAOTHeader()` before,  get `firstDescriptor.address == NULL`. I'm not sure why, maybe it's because `sharedClassConfig->findSharedData` hasn't been initialized at this point?
```cpp
   currentThread->javaVM->sharedClassConfig->findSharedData(currentThread, "J9AOTHeader", sizeof("J9AOTHeader") - 1, J9SHR_DATA_TYPE_AOTHEADER, FALSE, &firstDescriptor, NULL);
   if (!firstDescriptor.address)
      {
      strncat(buff, "null", buffSize);
      return;
      }
```

Reply:
OK, I see. It is because of this check: https://github.com/eclipse/openj9/blob/master/runtime/shared_common/shrinit.cpp#L1860. Then I am fine with putting all the new code here inside if (NULL != currentThread->javaVM->jitConfig) {}